push_notifications/conf/app.py

```diff
@@ -0,0 +1,357 @@
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.six import string_types
+from .base import BaseConfig, check_apns_certificate
+from ..settings import PUSH_NOTIFICATIONS_SETTINGS as SETTINGS
+
+"""
```

Comment:
How about all three? The README should at least contain a shoutout and link to the wiki. Where in the source would you imagine shipping this example config? Would dropping it at tests/settings/app_config.py make sense?